bender_navsensors/src/bender_navsensors.cpp

```diff
@@ -0,0 +1,52 @@
+#include <ros/ros.h>
+#include <mavros_msgs/CommandBool.h>
```

Comment:
@matthew-oberg @camronCollinsworth Can you please add `mavros_msgs` as a dependency in `package.xml` and `CMakeLists` so we won't run into problems in the future? Thanks!

Reply:
Yessir, sorry about the delay. Dependencies have been added. Let me know if there are any issues and I will get them fixed. Thanks for the feedback!